stl/inc/__msvc_string_view.hpp

```diff
@@ -702,7 +713,7 @@ class _String_bitmap<_Elem, false> { // _String_bitmap for wchar_t/unsigned shor
     bool _Matches[256] = {};
 };
 
-template <class _Traits, bool _Special = _Is_specialization_v<_Traits, char_traits>>
+template <class _Traits, bool _Special = _Is_implementation_handled_char_traits<_Traits>>

```

Comment:
I observe that you have proposed to eliminate the template parameter.
However, as we still have tail call with a different value in some functions, we can't.
Perhaps this still should be done as a follow-up enhancement, after all vectorization, including `_not_` is done, and we have no tail calls.

Reply:
Maybe I'll do this when last tail call with the different template argument is eliminated as a result of control flow enhancements in vectorization changes.